modules/textures/src/lib/parsers/parse-basis.js

```diff
@@ -22,11 +22,30 @@ const OutputFormat = {
 };
 
 export default async function parseBasis(data, options) {
-  const {BasisFile} = await loadBasisModule(options);
+  switch (options.basis.module) {
+    case 'encoder':
+      const fileConstructors = await loadBasisEncoderModule(options);
+      switch (options.basis.decoderFormat) {
+        case 'KTX2':
+          return parseKTX2File(fileConstructors.KTX2File, data, options);
+        case 'Basis':
+        default:
+          return parseBasisFile(fileConstructors.BasisFile, data, options);
+      }
+    case 'transcoder':
+    default:
+      const {BasisFile} = await loadBasisModule(options);
```

Comment:
Can we align naming of loadBasisModule and loadBasisEncoderModule?

Reply:
Done